_includes/partials/relationships.html

```
@@ -1,31 +1,31 @@
     <figcaption>Example: Kubernetes Cluster Role assigned to a Service Account</figcaption>
 </figure>
 
- <figure class="tabcontent sibling" id="sibling" data-fancybox="images" data-caption="Kubernetes Service and Deployment with matching label selectors" href="https://docs.meshery.io/assets/img/meshmodel/relationships/sibling_relationship.svg"><br>
-        <img  alt="Kubernetes Service and Deployment with matching label selectors" src="{{ site.baseurl }}/assets/images/features/relationships/sibling_relationship.svg">
-        <figcaption>Example: Kubernetes Service and Deployment with matching label selectors</figcaption>
+<figure class="tabcontent hierarchical" id="hierarchical" data-fancybox="images" data-caption="Example: Kubernetes Deployment with ConfigMap applied" href="{{ site.baseurl }}/assets/images/features/relationships/hierarchical_inventory_relationship.svg"><br>
+    <img alt="Example: Kubernetes Deployment with ConfigMap applied" src="{{ site.baseurl }}/assets/images/features/relationships/hierarchical_inventory_relationship.svg">
+    <figcaption>Example: Kubernetes Deployment with ConfigMap applied</figcaption>
+</figure>
+
+<figure class="tabcontent hierarchical" id="hierarchical" data-fancybox="images" data-caption=" Hierarchical - Parent: Namespace (Parent) and ConfigMap (child), Role (Child)" href="{{ site.baseurl }}/assets/images/features/relationships/hierarchical_parent_relationship.svg"><br>
```

HTMLHint (1.9.2)

[error] The id value [ hierarchical ] must be unique.

(id-unique)